build.gradle

```diff
@@ -72,12 +72,19 @@ dependencies {
 
     // Metalava isn't released yet. Check in its jar and explicitly track its transitive deps.
     metalava files('metalava.jar')
+    // noinspection GradleDependency
```

Comment:
Ah, I am not sure if you want to do this. Just turn the inspection off in your own IDE.

Reply:
Jake has a PR to disable lint warnings. I'd like to get the fixes for TypedArrayTest.kt checked in though. Could you remove the changes to build.gradle please?

Reply:
The test changes aren't really needed since it just leaks recycled objects out of their pool, not native resource handles that actually accumulate. The GC grabs the instances when they go out of scope and the pool will replenish itself as it gets drained.